# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/powerpc/powerpc-utils/no-adb-mouse-h.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./backlight.c.orig	2004-09-25 17:41:39.148876864 +0200
+++ ./backlight.c	2004-09-25 17:41:43.604199552 +0200
@@ -24,7 +24,6 @@
 #include <sys/types.h>
 #include <linux/ioctl.h>
 #include <linux/types.h>
-#include <linux/adb_mouse.h>
 #include <linux/cuda.h>
 #include <linux/adb.h>
 #include <linux/pmu.h>
--- ./trackpad.c.orig	2004-09-25 17:41:31.366060032 +0200
+++ ./trackpad.c	2004-09-25 17:40:58.445064784 +0200
@@ -18,7 +18,6 @@
 #include <unistd.h>
 #include <errno.h>
 #include <fcntl.h>
-#include <linux/adb_mouse.h>
 #include <linux/cuda.h>
 #include <linux/adb.h>
 
